common/output-model.cpp

```diff
@@ -994,22 +994,9 @@ void stream_dashboard::draw_dashboard(ux_window& win, rect& r)
     auto min_x = 0.f;
     auto max_x = 1.f;
     auto min_y = 0.f;
```

Comment:
Should check if ImGui functions needs the graph to be normalized to [0,1] range or can we work with actual values (removing divisions to normalize deque values)